crates/uv-fs/src/lib.rs

```diff
@@ -104,6 +104,26 @@ pub fn remove_symlink(path: impl AsRef<Path>) -> std::io::Result<()> {
     fs_err::remove_file(path.as_ref())
 }
 
+/// Create a symlink at `dst` pointing to `src` or, on Windows, copy `src` to `dst`.
+///
+/// If working with a directory, use [`replace_symlink`] instead which will use a junction on
+/// Windows.
+pub fn symlink_copy_fallback_file(
```

Comment:
Bitten once again that `replace_symlink` is only valid for directories on Windows, so I've added a new method for files. I plan to do some sort of refactor following this to make it hard to call the wrong thing here.